docker/mysql/db/db_upgrade/portal/Update_RMC_DB_2023052500-2023060500.sql

```diff
@@ -0,0 +1,21 @@
+START TRANSACTION;
+
+CREATE TABLE `process_query` (
+  `id` INT(11) NOT NULL AUTO_INCREMENT,
+  `userID` VARCHAR(50) DEFAULT NULL,
+  `url` TEXT DEFAULT NULL,
+  `lastProcess` INT(10) NOT NULL,
+  PRIMARY KEY (`id`)

```

Comment:
Unsure how big this table is.  But it's likely we want an index on more than the id.

Reply:
I could maybe throw something at url since that gets used for look up, I could also throw an unsigned on the last processes now that I am looking at it.

Reply:
I asked Pete to take a look at this, we will need to hold until his notes arrive.

Reply:
Pete and I went over this and cleaned this up, there were a few other things that I did not think were issues like BTREE and ollation which were added by HEIDI when I was exporting the table from that. I cleaned that up as well.